spec/requests/acuant_sdk_spec.rb

```diff
@@ -12,6 +12,11 @@
   it 'has a public directory for the alternate version' do
     expect(acuant_entries).to include(alternate_version)
   end
+
+  it 'has only 2 acuant versions' do
```

Comment:
The description here is slightly different than what the spec is actually testing.

```suggestion
  it 'has between 2 and 3 versions' do
```